.husky/pre-commit

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
+
+node ./generate-sandpack-config.mjs
+
+git add banner/angular/.sandpack/sandpack.config.json
```

Comment:
ideally this would be a pattern match so new examples don't require an update to this file